services/servo.md

```diff
@@ -44,3 +44,8 @@ The servo motor will stop rotating when it is trying to move a ``stall_torque``
     const response_speed?: u16.16 s/60° @ 0x181
 
 Time to move 60°.
+
+    ro volatile current_angle?: i16.16 ° @ 0x182
```

Comment:
this should be a reading register

Reply:
You can't have multiple reading registers in a spec. There's only one defined in _system:
 
> ro reading: i32 @ 0x101

Reply:
Oh, I see: 

>     rw angle: i16.16 ° { typical_min=-90, typical_max=90 } @ value

is not reading.

Reply:
Use reading instead of 0x182